frontend/libs/studio-components/src/components/StudioRedirectBox/StudioRedirectBox.tsx

```diff
@@ -1,19 +1,19 @@
-import React, { type ReactNode } from 'react';
+import React, { type ReactNode, type ReactElement } from 'react';
 import cn from 'classnames';
-import classes from './RedirectBox.module.css';
+import classes from './StudioRedirectBox.module.css';
 import { StudioLabelAsParagraph } from '@studio/components';
 
-export type RedirectBoxProps = {
+export type StudioRedirectBoxProps = {
   title: string;
   children: ReactNode;
   className?: string;
 };
 
-export const RedirectBox = ({
+export const StudioRedirectBox = ({
   title,
   children,
   className: givenClassName,
-}: RedirectBoxProps): React.ReactElement => {
+}: StudioRedirectBoxProps): ReactElement => {
   const className = cn(classes.wrapper, givenClassName);
```

Comment:
Should we use HTMLAttributes<HTMLDivElement> here? To ensure we stay consistent? Then we can need our "className" prop.

Reply:
I agree! 😄